packages/go/ein/ad.go

```diff
@@ -342,6 +342,17 @@ func ParseComputerMiscData(computer Computer) []IngestibleRelationship {
 		}
 	}
 
+	if computer.IsDC && computer.DomainSID != "" {
```

Comment:
did you want to append the DCFor edge to the list of edges that are in the comment for this function?